.appveyor.yml

```diff
@@ -10,17 +14,24 @@ cache:
 
 environment:
   matrix:
+    - nodejs_version: '10'
     - nodejs_version: '8'
     - nodejs_version: '6'
 
+matrix:
+  fast_finish: true
```

Comment:
This immediately finishes the build once one of the jobs fails on any of the Node.js platforms being tested. I would prefer running the build on all versions to be able to see if the failure is only for any particular version of Node.js.

Reply:
Disagree. AppVeyor is _only_ for Windows; primary testing is on Travis. If it fails for _any_, there's  a major problem that has to be addressed, but it should fail similarly for all anyway. Why wait when you **know** something has failed?

Reply:
Fair enough, I stand corrected.